ElectronClient/app.js

```diff
@@ -599,7 +599,7 @@ class Application extends BaseApplication {
 			label: _('Refresh templates'),
 			click: async () => {
 				const templates = await TemplateUtils.loadTemplates(Setting.value('templateDir'));
-
+				if (shim.isElectron()) Setting.setValue('availableTemplates', templates);
```

Comment:
You don't need to add shim.isElectron here because its in an electron only file. Some goes for later in this file.